src/components/Footer/Footer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Footer.css';
+import footerLinks from '../../data/footer-links.json';
+
+function Footer() {
+  return (
+    <footer className="Footer">
+      <div className="Footer-Inner">
+        <ul className="Footer-Nav">
+          {footerLinks.map((item, idx) => <li className="Footer-NavItem" key={idx}>
```

Comment:
try to use something that _isn't_ the index as a key (this is the default for when a key is omitted anyhow) - maybe `item.title`?

background: https://reactjs.org/docs/lists-and-keys.html#keys